interfaces/cython/cantera/ck2yaml.py

```diff
@@ -2095,14 +2056,33 @@ def convert_mech(input_file, thermo_file=None, transport_file=None,
             nReactions = len(parser.reactions) + sum(len(surf.reactions) for surf in parser.surfaces)
             print('Wrote YAML mechanism file to {0!r}.'.format(out_name))
             print('Mechanism contains {0} species and {1} reactions.'.format(len(parser.species_list), nReactions))
-        return surface_names
+        return parser, surface_names
+
+    def show_duplicate_reactions(self, error_message):
+        reactions = []
+        for line in error_message.split('\n'):
+            match = re.match('>.*# Reaction ([0-9]+)', line)
+            if match:
+                reactions.append(int(match.group(1))-1)
+
+        if len(reactions) != 2:
+            print(error_message)
+            return
+
+        equation = str(self.reactions[0])
+        lines = [self.reactions[i].line_number for i in reactions]
+        print('Undeclared duplicate reaction {}\nfound on lines {} and {} of '
+              ' the kinetics input file.'.format(equation, lines[0], lines[1]))
 
 
 def convert_mech(input_file, thermo_file=None, transport_file=None,
                  surface_file=None, phase_name='gas', extra_file=None,
                  out_name=None, quiet=False, permissive=None):
-    return Parser.convert_mech(input_file, thermo_file, transport_file, surface_file,
-                               phase_name, extra_file, out_name, quiet, permissive)
+    parser, surface_names = Parser.convert_mech(
```

Comment:
```suggestion
    _, surface_names = Parser.convert_mech(
```

Since `parser` is not used in this function